src/Stack.js

```diff
@@ -1,15 +1,18 @@
-// #8 intro routing/navigation and #9.
+// #11 Improve Gallery Layout for lge screen
+
 
 import React from 'react';
 import {Link} from 'react-router-dom';
 
 export default function Stack(props) {
     return(
-        <Link to='/details' className='item'>
+        <Link to={'/' + props.id} className='item'> 
```

Comment:
Could you please use back-ticks here?

Reply:
Is this what you are referring to?
<Link to={`/${'/' + props.id}`} className='item'>